Composer/scripts/l10n-extractJson.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const fs = require('fs');
+const path = require('path');
+
+const merge = require('lodash/merge');
+
+const { keep, transFn } = require('./l10nUtils');
+
+const L10N_FIELDS = ['label', 'description', 'title', 'subtitle'];
+
+const inDir = process.argv[2];
```

Comment:
Let's consider adding the [yargs](https://www.npmjs.com/package/yargs) package to use as we start to write more scripts.

Reply:
Oh, that's a nice one. Nothing I'm adding uses arguments in that complex a way, but I'll definitely consider it for future additions.